### PR TITLE
Add benchmark inference_tract

### PR DESCRIPTION
**Pull request by jlb6740**

### CODE REVIEW

**Comment by rahulchaphalkar** [comment]
Need to update `config.inc` with new sha, else shows `run invalid`.

File: build.sh [hunk 1]
```diff
@@ -29,6 +29,9 @@ echo "Revision:" $REVISION
 echo "Build Sha:" $BUILD_SHA "vs" $CURRENT_BUILD_SHA "(calculated)"
 echo ""
 
+# Run pre-docker build setup scripts
+(cd ./benchmarks/inference_tract/rust-benchmark && ./setup.sh)
```
Comment (rahulchaphalkar) [inline build.sh:33]: `setup.sh` doesn't exist at that directory. Is the benchmark silently failing (since it didn't download a model)?

**Comment by rahulchaphalkar** [comment]
@jlb6740 I see some commits, is this ready for review?

**Comment by jlb6740** [comment]
Dropping this PR in favor of: https://github.com/bytecodealliance/wasm-score/pull/37. Will address SHA comment there. The config question should already be addressed there .. I use a build.rs script instead.

**Pull request closed without merging**